### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by Shiara-Deudor**
##NOTES

-added landing page
-global colors created

### CODE REVIEW

File: src/layouts/header/header.scss [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    border: none;
+    border-color: transparent;
+  }
+
+  .logo {
+    font-size: 25px;
+    color: $fade_white;
+    letter-spacing: 2px;
+  }
+
```
Comment (mikco-hipe) [inline src/layouts/header/header.scss:70]: new line

File: .gitignore [hunk 2]
```diff
@@ -22,3 +22,4 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+.env
```
Comment (mikco-hipe) [inline .gitignore:25]: new line

Comment (karljerico) [inline .gitignore:25]: Trailing white space

File: src/layouts/header/header.scss (hunk 1 above)
Comment (karljerico) [inline src/layouts/header/header.scss:70]: Trailing white space

File: src/layouts/header/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                <li><a href=""><GiHamburgerMenu className='header-icon'/></a></li>
+            </ul>
+            <ul>
+                <li><ImBooks className='header-icon'/> <a href="#" className='logo'>bibliophile</a></li>
+            </ul>
+            <form class="form">
+                <button>
+                    <svg width="17" height="16" fill="none" xmlns="http://www.w3.org/2000/svg" role="img" aria-labelledby="search">
+                        <path d="M7.667 12.667A5.333 5.333 0 107.667 2a5.333 5.333 0 000 10.667zM14.334 14l-2.9-2.9" stroke="currentColor" stroke-width="1.333" stroke-linecap="round" stroke-linejoin="round"></path>
+                    </svg>
```
Comment (mikco-hipe) [inline src/layouts/header/Header.jsx:23]: you can store the svg files to assets folder similar to how you import images

File: vite.config.js [hunk 4]
```diff
@@ -1,10 +1,19 @@
 import { defineConfig } from 'vite'
 import react from '@vitejs/plugin-react'
 import * as path from 'path'
+import dotenv from 'dotenv'
+
+dotenv.config()
 
 // https://vitejs.dev/config/
```
Comment (mikco-hipe) [inline vite.config.js:8]: is this necessary?

**Review by mikco-hipe — CHANGES_REQUESTED** [review]

File: vite.config.js (hunk 4 above)
Comment (mikco-hipe) [reply vite.config.js:8]: the comment i mean

**Review by mikco-hipe — APPROVED** [review]
LGTM 💯

**Pull request merged**